experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -44,23 +46,28 @@ class Flowables {
   }
 
   template <typename T>
-  static Reference<Flowable<T>> just(std::initializer_list<T> list) {
-    auto lambda = [ list, it = list.begin() ](
+  static Reference<Flowable<T>> justN(std::initializer_list<T> list) {
+    std::vector<T> vec;
+    std::move(list.begin(), list.end(), std::back_inserter(vec));
+
+    size_t i = 0;
```

Comment:
I initially had it as `[..., i = 0]` in the lambda's capture list, but then got a signed vs unsigned warning when comparing `i` with `v.size()`, so I declared it up here as a `size_t`.

(I don't think I can do [..., size_t i = 0]` in a capture list, but I haven't tried....